english_style_guide.md

```diff
@@ -0,0 +1,97 @@
+_This document has been adapted from
+https://github.com/raywenderlich/english-style-guide_
+
+-------------------------------------------------------------------------------
+
+# English Style Guidelines
+
+This is an English style guide that Hack Club documentation should follow to
+stay consistent in how we refer to common things.
+
+**Table of Contents**
+
+- [English Style Guidelines](#english-style-guidelines)
+  - [Terminology and Capitalization](#terminology-and-capitalization)
+    - [General Guides](#general-guides)
+    - [List of Terminology](#list-of-terminology)
+  - [Punctuation](#punctuation)
+  - [Other Style Guidelines](#other-style-guidelines)
+
+## Terminology and Capitalization
+
+### General Guides
+
+**Company names** Company names are proper nouns and should be capitalized
+according to how the company capitalizes it's name.
+
+### List of Terminology
+
+**club leader** Not _Hack Club leader_ or _club director_.
+
+**console** Not _command line_, _command prompt_, or _terminal_.
+
+**double-click** Not _double click_.
+
+**filename**
+
+**Hack Club**
+
+**hacker**
+
+**OK** Don't use _Okay_ or _Ok_.
+
+**pop-up**
+
+**right-click**
+
+**URL**
+
+## Punctuation
+
+**All sentences should end with punctuation**, even if they end with
+inline-code, a URL, or a filename.
+
+_Example:_
+
+_Correct: Go to https://google.com._
+
+_Correct: Make sure the css selector is set to `#username`._
+
+_Incorrect: This will save to `index.html`_
+
+**Serial (Oxford) Commas**
+
+For simplicity the final comma in a list of elements (usually the one before the
+'and' or 'or') is left in the list.
+
+## Other Style Guidelines
+
+**bold** Titles that are to be omitted from a table of contents should be bolded
+instead of added to a header. Usually, repeated titles should be bolded.
+
+_Example:_
+
+```
+## Add an Image to The Website
+
+We can add an image to the site with an `<img>` tag. Unlike `<p>` tags, `<img>`
+tags don't have a closing tag.
+
+**Our code so far:**
+    <p>This is text</p>
+    <img href="example.com/icon.png"></img>
+```
+
+**italicize** Anything that needs emphasis should be italicized.
+
+**inline code**
+Use the inline code style for any names directly used in code. _Don't_ use it
```

Comment:
I actually think that using code blocks for things that aren't code (or code related) is an anti-pattern and should be revisited.